src/ontology/mondo-edit.obo

```diff
@@ -183764,6 +183768,7 @@ synonym: "congenital muscular dystrophy, Fukuyama type" RELATED [Orphanet:272]
 synonym: "FCMD" EXACT ABBREVIATION [Orphanet:272]
 synonym: "Fukuyama congenital muscular dystrophy" EXACT [OMIM:253800, Orphanet:272]
 synonym: "MDDGA4" EXACT ABBREVIATION [OMIM:253800]
+synonym: "muscle-eye-brain-FKTN related" EXACT CLINGEN_PREFERRED []
```

Comment:
could you please add a dbxref here:
https://clinicalgenome.org/affiliation/50061/